### PR TITLE
Sets up zero log for structuring logging and nicer prompts (Issue 48)

### PR DESCRIPTION
**Pull request by aleh-null**

### CODE REVIEW

File: cmd/hide/main.go [hunk 1]
```diff
@@ -15,52 +14,61 @@ import (
...
 
-const ProjectsDir = "hide-projects"
-const DefaultDotEnvPath = ".env"
+const (
+	ProjectsDir       = "hide-projects"
+	DefaultDotEnvPath = ".env"
+)
 
 func main() {
+	fmt.Print(Splash)
```
Comment (aleh-null) [inline cmd/hide/main.go:28]: Let's print the amazing Splash before writing any logs.

File: cmd/hide/main.go [hunk 2]
```diff
@@ -87,13 +95,22 @@ func main() {
...
 
 	if err := http.ListenAndServe(port, mux); err != nil {
-		fmt.Println("Error starting server")
-		panic(err)
+		log.Fatal().Err(err).Str("port", port).Msg("Failed to start server")
+	}
+}
+
+func setupLogger(debug bool) {
+	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: zerolog.TimeFormatUnix, NoColor: false}
```
Comment (aleh-null) [inline cmd/hide/main.go:109]: Provides infinite configurability for console formatting. This should be good enough for now.

File: cmd/hide/main.go [hunk 3]
```diff
@@ -15,52 +14,61 @@ import (
...
-		log.Println("Warning: DOCKER_USER and DOCKER_TOKEN environment variables are empty. This might cause problems when pulling images from Docker Hub.")
+		log.Warn().Msg("DOCKER_USER or DOCKER_TOKEN environment variables are empty. This might cause problems when pulling images from Docker Hub.")
 	}
 
 	mux := http.NewServeMux()
 	dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
-
 	if err != nil {
-		panic(err)
+		log.Fatal().Err(err).Msg("Cannot initialize docker client")
```
Comment (aleh-null) [inline cmd/hide/main.go:63]: Don't panic, bro ;)

**Review by artmoskvin — APPROVED** [review]

**Comment by artmoskvin** [comment]
Fixes #48

**Pull request merged**